README.md

```diff
@@ -20,6 +20,8 @@ Bürokratt Central Operations
 
 - If you made changes to data mapper, rebuild the image again using `docker build -t docker build -t datamapper .`
 
+- Ruuter configuration should be changed for all the endpoints to work. PUT and DELETE should be added to the allowedMethodTypes array. See more information https://github.com/buerokratt/Ruuter/blob/65889552329249665e48656ed866cdf99cda391f/samples/CONFIGURATION.md
```

Comment:
Linked to 

- https://github.com/buerokratt/Ruuter/issues/164
- https://github.com/buerokratt/Ruuter/issues/165